test/integration/cli/hookfiles-cli-test.js

```diff
@@ -533,7 +533,7 @@ describe('CLI', () => {
       });
     });
 
-    describe('when setting the log output level with -l', () => {
+    describe('when setting the log output level with --level', () => {
```

Comment:
If I understand correctly, you are changing the test for clarity and the old argument (`-l`) is still supported?

Are we removing test coverage for the `-l` option here? Perhaps worth having tests for both -l and --level

Reply:
It's only for clarity, `-l` stays. I take it for granted the `optimist` deals with the options correctly, so they're not tested in both versions, usually just with one. If the CLI was better isolated, why not, but at this point it just an overhead.